src/lib/cache_lib/impl.ml

```diff
@@ -357,7 +372,7 @@ let%test_module "cache_lib test instance" =
                    any unconsumption handler calls" =
       setup () ;
       let logger = Logger.null () in
-      with_cache ~logger ~f:(fun cache ->
+      with_cache ~logger ~cache_exceptions:false ~f:(fun cache ->
```

Comment:
why is `cache_exceptions` hardcoded to be `false` here and e.g. `true` in `catchup_scheduler`, while it seemed to be fully parameterized before ?

Reply:
In unit tests we use only `dev` profile, and it was the only profile with `true` value. This rewriting changes behavior for `dev` profile, of course, but we weren't really using it for anything but unit tests :D